packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/StakingBanner/StakingBanner.template.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { NavLink } from 'react-router-dom'
+import {
+  Button,
+  Flex,
+  IconCloseCircle,
+  IconRewardsCircle,
+  Padding,
+  PaletteColors,
+  SemanticColors
+} from '@blockchain-com/constellation'
+
+import { Image, Text } from 'blockchain-info-components'
+
+import { CloseLink } from '../styles'
+import { CardWrapper } from './StakingBanner.model'
+
+const StakingBanner = ({ onClickClose, rate }) => (
+  <CardWrapper>
+    <Padding horizontal={1} vertical={1.5}>
+      <Flex flexDirection='row' alignItems='center' justifyContent='space-between' gap={16}>
```

Comment:
Is this flex container needed? I has the default flex direction, reads space between but has only one item, and I think the align items can be set as `align-self` if needed in the flex below

Reply:
@mperdomo there are 2 items in this flex.